docs/src/misc.rst

```diff
@@ -886,3 +886,15 @@ is not complete.
     for the NUL terminator.
 
     .. versionadded:: 1.47.0
+
+.. c:function:: int uv_set_threadpool_size(unsingned int number)
```

Comment:
Should we use `size_t` ?

Reply:
Makes sense

Reply:
Nevermind, the code seems to like unsigned int instead.

Reply:
Your spin_threads function takes size_t